patches/api/0374-Added-EntityToggleSitEvent.patch

```diff
@@ -0,0 +1,78 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: KyGuy2002 <IEatBeans#1165>
+Date: Fri, 11 Mar 2022 15:33:10 +0000
+Subject: [PATCH] Added EntityToggleSitEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java b/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..10655cce559b3e0a04a5349c985770d63e996bdd
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/entity/EntityToggleSitEvent.java
+@@ -0,0 +1,66 @@
++package io.papermc.paper.event.entity;
++
++import org.bukkit.entity.Entity;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.entity.EntityEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Is called when an entity that implements Sittable changes its sitting state.
```

Comment:
this isn't true, panda's can sit, and the event isn't called for them since they don't extend TameableAnimal

Reply:
I updated the phrasing, and pandas now call the event as well.  Are there any other entities im forgetting that dont extend TamebaleAnimal?  Thanks